test/metahelper/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+
+	"knative.dev/test-infra/test/metahelper/util"
+)
+
+func main() {
+	getKeyOpt := flag.String("get", "", "get val for a key")
+	saveKeyOpt := flag.String("set", "", "save val for a key, must have --val supplied")
+	valOpt := flag.String("val", "", "val to be modified, only useful when --save is passed")
+	flag.Parse()
+	// Create with default path
+	c, err := util.NewClient("")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var res string
+	switch {
+	case *getKeyOpt != "" && *saveKeyOpt != "":
```

Comment:
The `*`s are annoying, how about we change `flag.String` to `flag.StringVar`?

Reply:
as discussed offline, we are not using arg[0], will explore StringVar